lib/sfdx.sh

```diff
@@ -1,21 +1,26 @@
 #!/usr/bin/env bash
 
+invokeCmd() {
```

Comment:
@cwallsfdc Should we move this into `lib/common.sh` so that we have it defined in one spot?

Reply:
Can do, but `sfdx.sh` uses.

Ideally, we move all SFDX invocations to `sfdx.sh` and then move `invokeCmd` there.